test/tw-1999.test.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+###############################################################################
+#
+# Copyright 2006 - 2021, Tomas Babej, Paul Beckingham, Federico Hernandez.
```

Comment:
This file is reappearing from ddd367232e2dbd7b7666a680e10af8a4094f3f03, so no update to the copyright.